.gitignore

```diff
@@ -55,6 +55,7 @@ doxygen/
 .vscode/c_cpp_properties.json
 .vscode/launch.json
 .vscode/tasks.json
+.vscode/settings.json
```

Comment:
Please revert this change. 
```suggestion
```

Reply:
Thanks, fixed in 61a855f785e52948c152849148cd094652e8fb3e